include/revng/Pipeline/Target.h

```diff
@@ -45,68 +43,42 @@ class ContainerBase;
 /// its extension.
 class Target {
 private:
+  using PathComponents = std::vector<std::string>;
   PathComponents Components;
```

Comment:
Can you take a look at how much effort would it be to switch to `Location`?
It'd be the Right Thing To Do™.

Reply:
Right! My bad.